### PR TITLE
Add preview to confirmation page

### PR DESCRIPTION
**Pull request by adammahmood**
### What is the context of this PR?
Adds ability to preview a question confirmation page

### How to review 
- Ensure tests pass
- Ensure you can preview a question confirmation page

### CODE REVIEW

**Comment by adammahmood** [comment]
[Previewing a playback page (3)](https://trello.com/c/tRva7dWZ/689-previewing-a-playback-page-3)

File: eq-author/src/App/questionPage/Preview/index.js [hunk 1]
```diff
@@ -64,25 +60,13 @@ export const UnwrappedPreviewPageRoute = ({ loading, data }) => {
...
-        <PageTitle>
-          {title ? (
-            <div dangerouslySetInnerHTML={{ __html: title }} />
-          ) : (
-            <Error data-test="no-title" large>
-              Missing Page Title
-            </Error>
-          )}
-        </PageTitle>
+        <PageTitle title={title} />
```
Comment (hamishtaplin) [inline eq-author/src/App/questionPage/Preview/index.js]: If we're turning this into a component, can it use `children` instead?

File: eq-author/src/App/questionConfirmation/Preview/index.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+const Replay = styled.div`
+  background-color: ${colors.darkBlue};
+  color: ${colors.white};
+  width: auto;
+  min-width: 20em;
+  display: inline-block;
+  overflow: hidden;
+  margin-top: 0;
+  margin-bottom: 2em;
```
Comment (hamishtaplin) [inline eq-author/src/App/questionConfirmation/Preview/index.js]:
```suggestion
  margin-bottom: 1em;
```

**Comment by adammahmood** [comment]
@rmccar Make sure you're on the correct branch `689-preview-confirmation-page`  (as there's a few which start with `689-` due to issues with merging the UI restructure).  Also try `yarn install` before running tests.

File: eq-author/src/App/questionPage/Preview/index.js (hunk 1 above)
Comment (adammahmood) [reply eq-author/src/App/questionPage/Preview/index.js]: It doesn't make sense in this case: 

```
const PageTitle = ({ title }) => {
  let pageTitle = title && title.replace(/(<p[^>]+?>|<p>|<\/p>)/gim, "");
  return (
    <Title>
      {pageTitle ? (
        /*  eslint-disable-next-line react/no-danger */
        <div dangerouslySetInnerHTML={{ __html: pageTitle }} />
      ) : (
        <Error data-test="no-title" large>
          Missing Page Title
        </Error>
      )}
    </Title>
  );
};

```

**Pull request merged**